### PR TITLE
Announce that the new official IRC channel is on Libera.Chat

### PR DESCRIPTION
**Pull request by antocuni**

### CODE REVIEW

**Review by hodgestar — APPROVED** [review]

**Comment by arigo** [comment]
Maybe write down explicitly ``irc.libera.chat`` somewhere?  Otherwise people have to follow even more links until they figure that out.

**Comment by antocuni** [comment]
> Maybe write down explicitly `irc.libera.chat` somewhere? Otherwise people have to follow even more links until they figure that out.

good idea. I added `irc://` links which connect directly to the server. They seem to work on my system with hexchat.

Unless anyone has objections, I'll merge the PR tonight or tomorrow morning

**Comment by mattip** [comment]
Parallel PR to pypy.org would also be welcome

File: posts/2021/05/hpy-irc-moves-to-libera-chat.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+-->
+
+Because of the
+[recent news about Freenode](https://arstechnica.com/gadgets/2021/05/freenode-irc-has-been-taken-over-by-the-crown-prince-of-korea/)
+and following the example of many other FOSS projects, the HPy team has
```
Comment (mattip) [inline posts/2021/05/hpy-irc-moves-to-libera-chat.md:15]: I would say either add a less inflammatory link or just remove the link:
```suggestion
Following the example of many other FOSS projects, the HPy team has
```

File: posts/2021/05/hpy-irc-moves-to-libera-chat.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+decided to move its official ``#hpy`` IRC channel from Freenode to
+[Libera.Chat](https://libera.chat/): [irc.libera.chat/hpy](irc://irc.libera.chat/hpy)
+
+The current channel on Freenode will soon be dismissed, so we recommend to
```
Comment (mattip) [inline posts/2021/05/hpy-irc-moves-to-libera-chat.md:19]:
```suggestion
The core devs will no longer be present on the Freenode channle, so we recommend to
```

File: posts/2021/05/hpy-irc-moves-to-libera-chat.md (hunk 1 above)
Comment (arigo) [reply posts/2021/05/hpy-irc-moves-to-libera-chat.md:15]: I'm +1 on just removing the link and stating as a fact that we're moving.

**Comment by antocuni** [comment]
> Parallel PR to pypy.org would also be welcome

done :) https://github.com/pypy/pypy.org/pull/27

**Pull request merged**